lib/YaST/workarounds.pm

```diff
@@ -48,4 +49,26 @@ sub apply_workaround_bsc1204176 {
     }
 }
 
+=head2 apply_workaround_bsc1207157 ():
+
+Workaround for the iscsi return code issue.
+
+Records a soft failure with a reference to bsc#1207157
+
+Changes the iscsid service file to require and start after the iscsid socket.
+Then reloads systemd, in order to scan for the changed unit.
+
+=cut
+
+sub apply_workaround_bsc1207157 {
+    record_soft_failure('bsc#1207157 - openQA test fails in iscsi_client - launched then no response');
```

Comment:
we should mention the ret code here, is the no response on launching related somehow? Perhaps is wrong in the bug as well because initially we don't install the dep? Let's fix bug title and sof-failure description if it is that the case.